# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/cyrus-sasl2/gcc40.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- cyrus-sasl-2.1.21/lib/client.c	2004-11-17 15:19:30.000000000 +0100
+++ src.cyrus-sasl2.1134357057.12621.2778701567/cyrus-sasl-2.1.21/lib/client.c	2005-12-12 04:22:56.000000000 +0100
@@ -61,8 +61,6 @@
 
 static cmech_list_t *cmechlist; /* global var which holds the list */
 
-static sasl_global_callbacks_t global_callbacks;
-
 static int _sasl_client_active = 0;
 
 static int init_mechlist()
